### PR TITLE
Add variance-related information to Distribution

### PR DESCRIPTION
**Pull request by a-veitch**
Add sum of squares of deviation from mean measure to Distribution, which can be used to calculate variance, standard deviations etc.

### CODE REVIEW

File: core/src/main/java/com/google/instrumentation/stats/Distribution.java [hunk 1]
```diff
@@ -103,6 +104,16 @@ final double getMean() {
...
 
   /**
+   * The sum of squared deviations from the mean for values in the population. For values x_i this
+   * is Sum[i=1..n]((x_i - mean)^2)
+   *
+   * <p>If {@link #getCount()} is zero then this value will also be zero.
+   *
+   * @return The sum of squared deviations from the mean for values in the population.
+   */
+  abstract double getSumSquaredDeviations();
```
Comment (sebright) [inline core/src/main/java/com/google/instrumentation/stats/Distribution.java]: Should this be `getSumOfSquaredDeviations`?

File: core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java [hunk 2]
```diff
@@ -95,6 +97,22 @@ long getCount() {
...
+   *
+   * <p>If {@link #getCount()} is zero then this value will also be zero.
+   *
+   * <p>Computed using Welfords method (see
+   * https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance, or Knuth, "The Art of
+   * Computer Programming", Vol. 2, page 323, 3rd edition
+   *
+   * @return The sum of squared deviations from the mean for values in the population.
+   */
+  double getSumSquaredDeviations() {
```
Comment (sebright) [inline core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java]: Same here, should it be getSumOfSquaredDeviations?

File: core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java [hunk 3]
```diff
@@ -95,6 +97,22 @@ long getCount() {
...
 
   /**
+   * The sum of squared deviations from the mean for values in the population. For values x_i this
+   * is Sum[i=1..n]((x_i - mean)^2)
+   *
+   * <p>If {@link #getCount()} is zero then this value will also be zero.
+   *
+   * <p>Computed using Welfords method (see
+   * https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance, or Knuth, "The Art of
+   * Computer Programming", Vol. 2, page 323, 3rd edition
```
Comment (sebright) [inline core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java]: The closing parenthesis is missing.

File: core/src/main/java/com/google/instrumentation/stats/Distribution.java (hunk 1 above)
Comment (a-veitch) [inline core/src/main/java/com/google/instrumentation/stats/Distribution.java]: Sure, done.

File: core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java (hunk 3 above)
Comment (a-veitch) [inline core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java]: Thanks, fixed.

File: core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java (hunk 2 above)
Comment (a-veitch) [inline core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java]: Done.

**Pull request merged**